src/cirrus/environment.py

```diff
@@ -24,12 +34,21 @@ def cirrus_home():
         return os.environ['CIRRUS_HOME']
     
     home = inspect.getsourcefile(cirrus)
-    # from the cirrus init py in the venv, we need to
-    # move up 5 dirs to get the install directory
-    for i in range(6): 
-        home = os.path.dirname(home)
+    if ('venv' in home) and ('site-packages' in home):
+        # we are in a pip installed virtualenv site-packages
+        # from the cirrus init py in the venv, we need to
+        # move up 5 dirs to get the install directory
```

Comment:
inspect gives me the location of the cirrus init py, then it just looks at the path. If it has lib/python*/site-packages in the path it has been pip installed.